doc/develop/west/basics.rst

```diff
@@ -6,14 +6,20 @@ Basics
 This page introduces west's basic concepts and provides references to further
 reading.
 
-West's built-in commands allow you to work with *projects* (Git
-repositories) under a common *workspace* directory.
+West's built-in commands allow you to work with :term:`projects <west project>`
+(Git repositories) under a common :term:`workspace <west workspace>` directory.
+
+West works in the following manner: the ``west init`` command creates the
+:term:`west workspace`, and clones the :term:`manifest repo <west manifest
+repository>`, while the ``west update`` command initially clones, and later updates, the
+:term:`projects <west project>` listed in the manifest in the workspace.
 
 Example workspace
 *****************
 
-If you've followed the upstream Zephyr getting started guide, your
-workspace looks like this:
+If you've followed the :ref:`getting_started`, your local
+:term:`west workspace`, which in this case is the folder named
+:file:`zephyrproject` as well as all its subfolders, looks like this:
 
```

Comment:
Just noticed that "net-tools" is misplaced in the snippet below. The whole folder example might be worth checking and cleaning up in a follow up PR to make sure it's somewhat aligned with how a current Zephyr workspace looks like

Reply:
Yes, will do.